Makefile.in

```diff
@@ -1,8 +1,9 @@
-px-objs = pxd.o dev.o iov_iter.o px_version.o io.o kiolib.o pxd_bio_makereq.o pxd_fastpath.o
+px-objs = pxd.o dev.o iov_iter.o px_version.o io.o kiolib.o pxd_bio_makereq.o pxd_bio_blkmq.o pxd_fastpath.o
 obj-m = px.o
 
 KBUILD_CPPFLAGS := -D__KERNEL__
-PXDEFINES := -D__PXKERNEL__ -D__PX_FASTPATH__ -D__PXD_BIO_MAKEREQ__
+#PXDEFINES := -D__PXKERNEL__ -D__PX_FASTPATH__ -D__PXD_BIO_MAKEREQ__
```

Comment:
Can remove?

Reply:
normally i would, this seems important not to forget, so wanted to leave it as a placeholder to know the switch.